src/navigators/Navigator.tsx

```diff
@@ -143,7 +151,7 @@ class Navigator extends React.Component<Props, State> {
   render() {
     return (
       <>
-        <AppStateManager handleAppComesToForeground={this.refreshWallets} />
+        <AppStateManager handleAppComesToForeground={this.refresh} />
```

Comment:
So it is 'on app comes to foreground' fetch not 'live' fetching?

Reply:
Live fetching will still work but when the app is in the background we won't get past events so we have to synchronize transactions and block height